week1-4/account.html

```diff
@@ -0,0 +1,143 @@
+                <div class="historylist-amount minus">-10800</div>
+                <div class="historylist-delete">x</div>
+        </section>
+        <section class="historylist-items">
+            <div class="historylist-category">생활비</div>
+            <div class="historylist-contents">다이소</div>
+            <div class="historylist-amount minus">-3000</div>
+            <div class="historylist-delete">x</div>
+        </section>
+        <section class="historylist-items">
+            <div class="historylist-category">식비</div>
+            <div class="historylist-contents">표표마라탕 신림점</div>
+            <div class="historylist-amount minus">-12000</div>
+            <div class="historylist-delete">x</div>
+        </section>
+        <section class="historylist-items">
+            <div class="historylist-category">알바비</div>
+            <div class="historylist-contents">미술학원 알바비</div>
+            <div class="historylist-amount plus">+200000</div>
+            <div class="historylist-delete">x</div>
+        </section>
+        <section class="historylist-items">
+            <div class="historylist-category">식비</div>
+            <div class="historylist-contents">이것은 갈비인가 통닭인가 안녕하세요 수원왕갈비 통닭입니다</div>
+            <div class="historylist-amount minus">-13000</div>
+            <div class="historylist-delete">x</div>
+        </section>
+        <section class="historylist-items">
+            <div class="historylist-category">식비</div>
+            <div class="historylist-contents">모모유부 홍대점</div>
+            <div class="historylist-amount minus">-10000</div>
+            <div class="historylist-delete">x</div>
+        </section>
+        <section class="historylist-items">
+            <div class="historylist-category">식비</div>
+            <div class="historylist-contents">버거킹 문정역점</div>
+            <div class="historylist-amount minus">-10800</div>
+            <div class="historylist-delete">x</div>
+        </section>
+
+        <section class="historylist-items">
+            <div class="historylist-category">식비</div>
+            <div class="historylist-contents">마녀김밥 서울대입구점</div>
+            <div class="historylist-amount minus">-8000원</div>
+            <div class="historylist-delete">x</div>
+        </section>
+        <section class="historylist-items">
+            <div class="historylist-category">식비</div>
+            <div class="historylist-contents">버거킹 문정역점</div>
+            <div class="historylist-amount minus">-10800</div>
+            <div class="historylist-delete">x</div>
+        </section>
+        <section class="historylist-items">
+            <div class="historylist-category"></div>
+            <div class="historylist-contents"></div>
+            <div class="historylist-amount minus"></div>
+            <div class="historylist-delete"></div>
+        </section>
+    </section>
+
+    <footer>
+        <div class="button" onclick="click">+</div>
```

Comment:
여기도 button 태그를 사용해주세요~! 

더불어,
button 태그는 reset, button, submit 세가지의 type을 가져요!

default는 submit이므로, <button type="button"> 이렇게 type을 button으로 명시해주어야 합니당!

[버튼에 타입을 쓰는 이유](https://nykim.work/96)
```suggestion
        <button type="button">+</button>
```